fuzz/fuzz_targets/fuzz_client.rs

```diff
@@ -0,0 +1,32 @@
+#![no_main]
+use h2_support::prelude::*;
+use libfuzzer_sys::{arbitrary::Arbitrary, fuzz_target};
+
+#[derive(Debug, Arbitrary)]
+struct HttpSpec {
+    uri: Vec<u8>,
+    header_name: Vec<u8>,
+    header_value: Vec<u8>,
+}
+
+async fn fuzz_entry(inp: HttpSpec) {
+    if let Ok(req) = Request::builder()
+        .uri(&inp.uri[..])
+        .header(&inp.header_name[..], &inp.header_value[..])
+        .body(())
+    {
+        let (io, mut _srv) = mock::new();
+        let (mut client, _h2) = client::Builder::new()
+            .initial_stream_id(::std::u32::MAX >> 1)
```

Comment:
I don't think this is really needed (I mean, unless you wanted to fuzz the handling of this option, but I'm not worried about that really...).

Reply:
Thanks - should be fixed now!